brainscore/lookup.csv

```diff
@@ -1,117 +1,121 @@
+dicarlo.Marques2020_size,stimulus_set,StimulusSet,S3,https://brainio.dicarlo.s3.amazonaws.com/image_dicarlo_Marques2020_size.csv,0fd0aeea8fa6ff2b30ee9a6a684d4600590d631f,,brainio_brainscore
+dicarlo.Marques2020_size,stimulus_set,,S3,https://brainio.dicarlo.s3.amazonaws.com/image_dicarlo_Marques2020_size.zip,11122de2784218b7dd4cd0b557e62583ca7ee283,,brainio_brainscore
+movshon.FreemanZiemba2013_properties,stimulus_set,StimulusSet,S3,https://brainio.contrib.s3.amazonaws.com/image_movshon_FreemanZiemba2013_properties.csv,6f7bcb5d0c01e81c9fbdcec9bf586cbb579a9b02,,brainio_brainscore
+movshon.FreemanZiemba2013_properties,stimulus_set,,S3,https://brainio.contrib.s3.amazonaws.com/image_movshon_FreemanZiemba2013_properties.zip,c64a0d6b447676a6d5cff3d3625a5f294e62ff97,,brainio_brainscore
+movshon.Cavanaugh2002a,assembly,PropertyAssembly,S3,https://brainio.contrib.s3.amazonaws.com/assy_movshon_Cavanaugh2002a.nc,d4b24f4bbf8a14138a8e98391ab03796c2c05e7d,dicarlo.Marques2020_size,brainio_brainscore
+devalois.DeValois1982a,assembly,PropertyAssembly,S3,https://brainio.contrib.s3.amazonaws.com/assy_devalois_DeValois1982a.nc,8f4fd70e987c3c566b5b6c6bf15ef27637618b2c,dicarlo.Marques2020_orientation,brainio_brainscore
+devalois.DeValois1982b,assembly,PropertyAssembly,S3,https://brainio.contrib.s3.amazonaws.com/assy_devalois_DeValois1982b.nc,611176eefdac09feaa4c07f081784c24067629e1,dicarlo.Marques2020_spatial_frequency,brainio_brainscore
+movshon.FreemanZiemba2013_V1_properties,assembly,PropertyAssembly,S3,https://brainio.contrib.s3.amazonaws.com/assy_movshon_FreemanZiemba2013_V1_properties.nc,e665522d9a32e6cd6c3ebc61ff65fd3899c8d3e6,movshon.FreemanZiemba2013_properties,brainio_brainscore
+shapley.Ringach2002,assembly,PropertyAssembly,S3,https://brainio.contrib.s3.amazonaws.com/assy_shapley_Ringach2002.nc,0f3e19140bd9f930109879a38e574471d9576cf5,dicarlo.Marques2020_orientation,brainio_brainscore
+schiller.Schiller1976c,assembly,PropertyAssembly,S3,https://brainio.contrib.s3.amazonaws.com/assy_schiller_Schiller1976c.nc,58daa02ba2680e75dcb11b37eb6085c4afb6576e,dicarlo.Marques2020_spatial_frequency,brainio_brainscore
+Geirhos2021_colour,stimulus_set,StimulusSet,S3,https://brainio-brainscore.s3.amazonaws.com/image_Geirhos2021_colour.csv,ea66ce8a750a93b0b2238dd93fc4c0dd085454af,,brainio_brainscore
+Geirhos2021_colour,stimulus_set,,S3,https://brainio-brainscore.s3.amazonaws.com/image_Geirhos2021_colour.zip,2d0386e74d164f41354d65ba3b865b398c56aece,,brainio_brainscore
+Geirhos2021_contrast,stimulus_set,StimulusSet,S3,https://brainio-brainscore.s3.amazonaws.com/image_Geirhos2021_contrast.csv,1727b77c5abd5c15a660e82494c60bcd23381989,,brainio_brainscore
+Geirhos2021_contrast,stimulus_set,,S3,https://brainio-brainscore.s3.amazonaws.com/image_Geirhos2021_contrast.zip,48b6c7cb9d26bf87b99a7d87a9331e071fb1ca70,,brainio_brainscore
```

Comment:
current convention is `<lab name>.<first author(s) name(s)><year>`
so this should be `brendel.Geirhos2021_{colour,contrast}`